### PR TITLE
Countdown

### PR DESCRIPTION
**Pull request by muskaanshah**
Features added: 
User can add a countdown
Event details stored in localstorage
Event gets deleted after it's over
User can edit / delete the event

### CODE REVIEW

File: src/components/countdown/Countdown.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+				description: "",
+				dateByUser: "",
+		  }
+		: {
+				...JSON.parse(localStorage.getItem("Event")),
+				dateByUser: dateFormat(
+					new Date(JSON.parse(localStorage.getItem("Event")).dateByUser)
+				),
+		  };
+	console.log(eventFromLocalStorage);
```
Comment (Pratik1005) [inline src/components/countdown/Countdown.jsx:19]: remove console log

**Review by Pratik1005 — APPROVED** [review]
Countdown is working, very good UI. 
Just a suggestion: when we visit the site the input name field can be in focus so that we can directly write our name.

File: src/components/countdown/Countdown.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import { useEffect, useState } from "react";
+
+import "./countdown.css";
+import { CountdownModal } from "./CountdownModal";
+import { dateFormat } from "./dateFormat";
+
+function Countdown() {
+	const eventFromLocalStorage = !localStorage.getItem("Event")
+		? {
```
Comment (JowelTisso) [inline src/components/countdown/Countdown.jsx:9]: You can extract localStorage function separately

**Review by JowelTisso — APPROVED** [review]
* Check your code indentation
* Rest Looks Good

**Review by manismk — APPROVED** [review]
Code is clean and functionality works properly

**Pull request merged**